<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PestanaPrincipal</class>
 <widget class="QMainWindow" name="PestanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>459</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>158</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>158</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>72</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Identificador de Piezas</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::Scanner">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QGroupBox" name="VentanaDeInput">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="ImagenPath">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;FreeMono&quot;;
color: #ff9e3d;
background-color: #111822;
border: 1px solid #ff9e3d;
padding: 2px 4px;
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>/home/barrios14101/Documents/GitHub/IA/Database/AV/Image.jpg</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="BotonImagen">
         <property name="toolTip">
          <string>Seleccionar una única imagen para análisis</string>
         </property>
         <property name="text">
          <string>Analizar Imagen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="CarpetaPath">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;FreeMono&quot;;
color: #ff9e3d;
background-color: #111822;
border: 1px solid #ff9e3d;
padding: 2px 4px;
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>/home/barrios14101/Documents/GitHub/IA/Database/AV</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="BotonCarpeta">
         <property name="toolTip">
          <string>Seleccionar carpeta con imágenes a analizar</string>
         </property>
         <property name="text">
          <string>Analizar Carpeta</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCommandLinkButton" name="BotonGrabar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Iniciar grabación del módulo de Reconocimiento de Voz</string>
         </property>
         <property name="text">
          <string>Comenzar Grabación</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::MediaRecord">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QProgressBar" name="BarraCargando">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>158</green>
               <blue>61</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>158</green>
               <blue>61</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>65</red>
               <green>72</green>
               <blue>78</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="ComandoLabel">
            <property name="text">
             <string>Comando identificado: ---</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BotonConfirmarCmd">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2e7d32;
    color: white;
    border-radius: 4px;
    padding: 4px 8px;
}
QPushButton:hover {
    background-color: #388e3c;
}
</string>
            </property>
            <property name="text">
             <string>Confirmar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BotonReproducirCmd">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #c62828;
    color: white;
    border-radius: 4px;
    padding: 4px 8px;
}
QPushButton:hover {
    background-color: #d32f2f;
}
</string>
            </property>
            <property name="text">
             <string>Reproducir</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Graficos">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="DataFrame" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="KMeansGraph" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="BayesGraph" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuPrincipal">
    <property name="title">
     <string>Principal</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAV_Debug">
    <property name="title">
     <string>AV Debug</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVR_Debug">
    <property name="title">
     <string>VR Debug</string>
    </property>
   </widget>
   <addaction name="menuPrincipal"/>
   <addaction name="menuAV_Debug"/>
   <addaction name="menuVR_Debug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BotonImagen</tabstop>
  <tabstop>BotonCarpeta</tabstop>
  <tabstop>BotonGrabar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
